app/src/main/res/layout/activity_login.xml

```diff
@@ -117,6 +117,18 @@
                     android:textStyle="bold" />
             </LinearLayout>
 
+            <TextView
+                android:id="@+id/txt_login_helper"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_gravity="center"
+                android:gravity="center"
+                android:paddingTop="@dimen/padding_short"
+                android:paddingBottom="4dp"
+                android:textColor="@color/green_500"
```

Comment:
I would change the color to grey, as this is not a success message.

Reply:
Do you want me to do that?
Also a suggestion, maybe red color can be used as this is a information we would like to highlight before a user tries to log in.

Reply:
@teolemon what do you think?

Reply:
red and green convey strong meaning. Green rather than Red in the sense that the user hasn't done anything wrong. Grey is an option as well, but it might not be as visible ?

Anyway, we don't need to overthink it too much in terms of priorities right now (it's going to be on OBF, OPF and OPF vs OFF)

Reply:
so merging as is or after changing to grey are both fine to me

Reply:
I will change it to grey.

Reply:
I have made the changes.

Reply:
@VaiTon